app/packages/core/src/components/Sidebar/Entries/GroupEntries.tsx

```diff
@@ -25,7 +25,12 @@ import { PillButton } from "../../utils";
 
 import { useTheme } from "@fiftyone/components";
 import Draggable from "./Draggable";
-import { getDatasetName } from "@fiftyone/state";
+import {
+  datasetName,
+  getDatasetName,
```

Comment:
Is this still used? Need to avoid depending on `context` as its not available in the embed form factor. It looks like you only depend on `datasetName` which I think it set properly in the embed wrapper: `<Dataset />`.